src/libs/actions/App.js

```diff
@@ -114,10 +115,11 @@ function getAppData(shouldSyncPolicyList = true, shouldSyncVBA = true) {
     }
 
     // We should update the syncing indicator when personal details and reports are both done fetching.
-    return Promise.all([
+    return promiseAllSettled([
```

Comment:
Just updated this to `promiseAllSettled` not because I expect one of these to fail or know a case where they might, only because it makes sense to me that even if they did fail for some reason we don't know about right now, we would still want to disable the loading spinner by setting `IS_LOADING_AFTER_RECONNECT` to false.

Reply:
Maybe we could achieve this same thing with `Promise.all` and `.finally`? Not sure which is better or that it matters which we use.

Reply:
Switched back to Promise.all